src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -162,12 +162,14 @@ TypeAnnotations BuildTypeAnnotations (TypeDefinition type)
 								paramAnnotations[0] = methodMemberTypes;
 							}
 						} else if (methodMemberTypes != DynamicallyAccessedMemberTypes.None) {
-							_context.LogWarning ($"The DynamicallyAccessedMembersAttribute is only allowed on method parameters, return value or generic parameters.", 2041, method);
+							_context.LogWarning ($"The DynamicallyAccessedMembersAttribute is only allowed on method parameters, return value or generic parameters.",
+								2041, method, subcategory: MessageSubCategory.TrimCorrectness);
```

Comment:
I would argue that this warning should not be part of the TrimCorrectness subcategory. It warns about invalid usage of the attribute - in itself it doesn't affect correctness. In my mind this is similar to invalid XML in the descriptor.